README.md

```diff
@@ -1,6 +0,0 @@
-Addons for Odoo by Therp BV
```

Comment:
Why this deletion?

Reply:
Well, this as a sentence does not make much sense now that we have a readme folder explaining everything. Of course this module is an addon for Odoo and of course the author is Therp BV as written in the manifest

Reply:
Yes but this is not for an addon, but an indication of what the repository is for.

Reply:
Ok I missed this and I thought it was a addon readme while it was for the repo. I reverted this. Please let me know if everything is ok so I can squash